src/scripts/containers/VideoManagerContainer.js

```diff
@@ -1,16 +1,21 @@
+import { Map } from 'immutable'
```

Comment:
FYI in the rest of the repo we have been keeping the `components/` files separate from the `containers/` files, rather than merging them into one. We can go either way, but we should try to be consistent.

Keeping them separate does make it a bit easier to _unit_ test, but that's not something we're doing extensively right now.